dspace-spring-rest/src/test/java/org/dspace/app/rest/BitstreamRestRepositoryIT.java

```diff
@@ -347,7 +347,8 @@ public void findOneBitstreamRelsTest() throws Exception {
 
 
         getClient().perform(get("/api/core/bitstreams/" + bitstream.getID() + "/self"))
-                   .andExpect(status().isOk())
+                   //Does this status make sense?  This is what is being returned.
```

Comment:
no this has no sense... this endpoint doesn't exist at all so a 404 should be returned. Nevertheless it is a separate issue. Can you remove the check of the /self and fire a bug report?